.rubocop.yml

```diff
@@ -0,0 +1,12 @@
+inherit_from: 
+  - .rubocop_todo.yml
+
+AllCops:
+  Include:
+    - Rakefile
+
+StringLiterals:
+  Enabled: false
+
+LineLength:
+  Enabled: false
```

Comment:
Based on a cursory observation, oxidized has a preference for verbose style hash methods (in terms of instances in the codebase), perhaps something along these lines as well?

```yaml
# Stick to verbose until https://bugs.ruby-lang.org/issues/10177 is closed.
Style/PreferredHashMethods:
  EnforcedStyle: verbose
```